### ISSUE TITLE
Container element is empty (menu)

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9475]
# Motivation

Siteimprove is reporting a WCAG 2.1 A accessibility issue for Container element is empty.

Some roles are designed to contain other roles. This element has this type of role, but it doesn't contain any [required-owned elements](https://www.w3.org/TR/wai-aria-1.1/#mustContain).

# Additional Info

We addressed this issue previously as part of https://github.com/uiowa/uiowa/issues/9254

Being reported on 58 SiteNow sites.

This appears to be related to the menu system and specifically when there are child menu items.

Obermann Center example - https://my2.siteimprove.com/Inspector/1006434/A11y/Page?pageId=33370252600&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r68/failed/L3BXV0p0eEY=;MTMwNzBiMjI=

SHL example - https://my2.siteimprove.com/Inspector/1218676/A11y/Page?pageId=77670586839&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r68/failed/bmdFTiszUms=;Y2UxYmI4Y2E=

**Comment by briand44** [comment]
## Grooming
 
**Quadrant:** 

Complex - previous solution may not apply directly to this issue using different components

 **Discussion:**
 - seems like a mega menu related issue
 - looks like this mega menu uses a different template than what we updated previously
 - pull site down, debug to identify template, compare to previous solution in other template
 - check to see if toggle navigation reports this issue as well

**Proposed solution or definition of done:** 
- Address this issue for mega menu similar to previous fix
- Check to see if issue exists in toggle navigation and address there as well if so
  - https://writing.prod.drupal.uiowa.edu/  probably only site out there using toggle nav.
- verify Siteimprove no longer reporting
